### PR TITLE
Feature/design system basics

### PR DESCRIPTION
**Pull request by ankush29**

### CODE REVIEW

**Comment by ankush29** [comment]
test case : pending
table for all component : pending (work in progress)

File: lib/components/molecules/Table/Table.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  data: Array,
+};
+
+const Table = ({ data, className, ...others }: tableProps): Node => {
+  const columnName = Object.keys(data[0]);
+
+  const headers = (
+    <tr>
+      {columnName.map((name, n) => (
+        // eslint-disable-next-line react/no-array-index-key
```
Comment (vinodloha) [inline lib/components/molecules/Table/Table.js:24]: This style of disabling errors is not appreciated, these checks have been configured for a purpose

File: lib/components/molecules/Table/Table.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <tr key={i}>
+      {columnName.map((prop, p) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <td key={p}>{item[prop]}</td>
+      ))}
+    </tr>
+  ));
+
+  return (
+    <table className={className} {...others}>
```
Comment (vinodloha) [inline lib/components/molecules/Table/Table.js:41]: This is very very less thoughtful implementation

File: lib/components/molecules/Table/Table.story.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import defaultConfig from './Table.mock';
+
+// Import Styled Component to showcase variations
+import Table, { TableVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+// $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code
+storiesOf('Molecules', module)
+  .addParameters({ jest: ['Table', 'TableVanilla'] })
+  .add('CTA Knobs', () => <TableVanilla {...defaultConfig} />)
```
Comment (vinodloha) [inline lib/components/molecules/Table/Table.story.js:13]: Remove the story title

File: lib/components/molecules/Table/Table.story.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './Table.mock';
+
+// Import Styled Component to showcase variations
+import Table, { TableVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+// $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code
```
Comment (vinodloha) [inline lib/components/molecules/Table/Table.story.js:10]: Delete this line

File: lib/components/molecules/Table/Table.style.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { css } from 'styled-components';
+
+export default css`
+  border-spacing: 0;
+  border: 1px solid black;
```
Comment (vinodloha) [inline lib/components/molecules/Table/Table.style.js:5]: Why you did not applied the Design system tokens?

File: lib/components/molecules/Table/tests/Table.test.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import { shallow } from 'enzyme';
+
+import Table from '../index';
+import defaultConfig from '../Table.mock';
+
+describe('<Table />', () => {
+  const TableComponent = shallow(<Table {...defaultConfig}>Test</Table>);
+  const table = TableComponent.find('table');
```
Comment (vinodloha) [inline lib/components/molecules/Table/tests/Table.test.js:8]: Whats the purpose of these variable?

**Pull request closed without merging**